wurst/closures/Execute.wurst

```diff
@@ -0,0 +1,32 @@
+package Execute
+
+public interface ForForceCallback
+	function callback()
+
+force dummy = CreateForce()
```

Comment:
You can use `bj_FORCE_ALL_PLAYERS` instead of a dummy

Reply:
Unfortunately, to execute the callback exactly once, there needs to be only one player inside the Force. I think using bj_FORCE_ALL_PLAYERS would cause it to execute multiple times instead.

Reply:
use `let` or `constant`, and `dumy` isn't a suitable name

Reply:
Thanks